xchange-stream-bankera/src/main/java/info/bitrich/xchangestream/bankera/BankeraStreamingService.java

```diff
@@ -26,7 +26,7 @@ public String getSubscribeMessage(String channelName, Object... args) throws IOE
   }
 
   @Override
-  public String getUnsubscribeMessage(String channelName) throws IOException {
+  public String getUnsubscribeMessage(String channelName, Object... args) throws IOException {
```

Comment:
can you break out changing this signature into a separate review?

Reply:
Done.